test/e2e/assets.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+type assets struct {
+	t              *testing.T
+	createdFolders []string
+}
+
+func (a assets) simpleAppDir() string {
```

Comment:
even though assets object is private to the package, its methods arguably should be public (ie public to users within this package). otherwise it reads like eveything is accessing privates of the test supporting objects.